### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by kvie**
Still to-do with this draft:
1. Add link regarding Ada's CB Connect contribution.
2. Add newsletter contributors.

### CODE REVIEW

File: august2019.md [hunk 1]
```diff
@@ -8,65 +8,61 @@ We have a friendly Slack community of knowledge sharers, and we let anyone on ou
...
+>
+> * Ask for feedback religiously. The interviewing process is nerve straining, but you will live. They are just people. No one likes the idea of rejection, but ask for feedback on how to improve. If you are lucky, they will give you feedback which is helpful.
+>
+> * Measure and record data - who did you talk to, which companies, what contacts, what dates/times, what subjects did you talk about? Did they mention other people? Write down everything and try to make a system to record it. Also track outcomes for ideas you have of what you think will or won’t work.
+>
+> * Continue to grow - make projects, learn about creating and communicating the value you create, learn about testing/validation, so on.
+
+2. Shout-out to @Ada for coming up with some amazingly beautiful designs for CB Connect in [link].
```
Comment (lpatmo) [inline august2019.md]: Please link to https://sketch.cloud/s/PKPgL! :)

**Review by lpatmo — CHANGES_REQUESTED** [review]
Thank you so much, @kvie!! One small change requested (link to Ada's designs). Thank you!

File: august2019.md (hunk 1 above)
Comment (kvie) [reply august2019.md]: Done!

**Review by lpatmo — APPROVED** [review]

**Comment by lpatmo** [comment]
Thanks so much @kvie!!

**Pull request merged**